March 2025 update (#60)
Ammalgam-Protocol/docs: docs/updates/2025/March.md

---
sidebar_position: 3
---

# March

## tl;dr

- We have preliminary reports from both auditors
- Considering audit results we are considering a small audit competition
- We are aiming for May Mainnet release, but the above item would delay us 

>   to June. 

Reviewer: xor add '.' above

```suggestion
  to June
```

- We will be releasing a Beta release of the front end soon in the next few 

>   weeks. 

Reviewer: xor add '.' above

```suggestion
  weeks
```

## Development

We have preliminary audit reports from both auditors. We are about 60% of 
the way through the fixes. We are aiming to complete them this month before 
having the auditors review. I created [this page](https://www.notion.

> so/Changes-Post-Audit-1d23195c2bf58019a357eba69861f832?pvs=21) to give the 

Reviewer: link broken

auditors a snapshot of where we are at. We want to finish changes by the end 
of this month before getting a final review from the auditors.

It was both fortunate to have two auditors who were able to find critical 
issues that we missed, and unfortunate that each auditor found a distinct 
critical issue that the other auditor did not find. I want to have the 
confidence that those are the only two issues, but taking a moment to triple 
check by considering a code audit competition.    

On the front end, our goal is to release the beta version of the app by the 
end of the month. Rebuilding validation on the front end has been slower 
than I had hoped due to complexity. We want to have a smooth user experience 
without failed transactions due to gaps between the front end and the 
contracts before we release a beta version.

## Go to Market

> We have been in contact with a number of chains, LP’s, Apps, investors, and 

Reviewer: ```suggestion
We have been in contact with a number of chains, LPs, Apps, investors, and 
```

curators and will continue outreach to the community to make sure everyone 
knows we are coming. With our incentive plan for early adopters finalized, 
we are working to distribute the terms through various channels.

We plan to have enough progress with LP conversations before we launch. Next 
month we should have a better idea of how we are progressing.

> - Worked to consolidate the communities growth, improving user experience 

Reviewer: make all points end with '.' xor not

  and server security in Discord. 
- Hosted our first Discord AMA with 30+ live attendees throughout, which 

>   we’ll be doing bi-weekly going forwards. 

Reviewer: ```suggestion
  we’ll be doing bi-weekly going forward. 
```

> - Began producing content for our blog

Reviewer 2: ```suggestion
- Began producing content for our blog.
```

> - Launched our first ambassador quests utilizing Zealy

Reviewer 2: ```suggestion
- Launched our first ambassador quests utilizing Zealy.
```

> - Reached 150+ total ambassadors

Reviewer 2: ```suggestion
- Reached 150+ total ambassadors.
```

> - Announced the completion of our first audit with Macro

Reviewer 2: ```suggestion
- Announced the completion of our first audit with Macro.
```
